README.md

```diff
@@ -1,4 +1,46 @@
-# Holly Sh.t!
+[![Kottans-Frontend](https://img.shields.io/badge/%3D%28%5E.%5E%29%3D-frontend-yellow.svg)](https://github.com/kottans/frontend)
```

Comment:
Hey, Vitaliy! There's clearly something wrong going on here. Please check "Files changed" tab and make sure you're following instructions of task right, because right now you're trying to merge patch-restore-readme-intro branch to master branch.